validator/keymanager/remote-web3signer/v1/requests.go

```diff
@@ -0,0 +1,292 @@
+package v1
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/pkg/errors"
+	validatorpb "github.com/prysmaticlabs/prysm/proto/prysm/v1alpha1/validator-client"
+)
+
+// GetBlockSignRequest maps the request for signing type BLOCK.
+func GetBlockSignRequest(request *validatorpb.SignRequest, genesisValidatorsRoot []byte) (*BlockSignRequest, error) {
```

Comment:
Add a check for making sure the genesis validators root len is > 32. No need to nil check, as only doing len(genesisValidatorsRoot) != 32 { return err } will suffice